cms/management/commands/startcmsproject.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+import os
+import subprocess
+import sys
+
+from django.core.checks.security.base import SECRET_KEY_INSECURE_PREFIX
+from django.core.management import CommandError
+from django.core.management.templates import TemplateCommand
+from django.core.management.utils import get_random_secret_key
+
+from cms import __version__ as cms_version
+
+
+class Command(TemplateCommand):
+    help = (
+        "Creates a django CMS project directory structure for the given project "
+        "name in the current directory or optionally in the given directory."
+    )
+    missing_args_message = "You must provide a project name."
+
+    def get_default_template(self):
+        from cms import __version__
+
+        version = ".".join(__version__.split(".")[:2])  # get major.minor
+        return f"https://github.com/django-cms/cms-template/archive/{version}.tar.gz"
+
+    def postprocess(self, project):
+        self.HEADING = self.style.SQL_FIELD
+        self.install_requirements(project)
+        os.chdir(project)
+        self.stdout.write(self.HEADING("Create migrations"))
+        self.run_management_command(["migrate"], capture_output=True)
+        self.stdout.write(self.HEADING("Create superuser"))
+        self.run_management_command(["createsuperuser"])
+        self.stdout.write(self.HEADING("Check installation"))
+        self.run_management_command(["cms", "check"])
+
+        message = f"django CMS {cms_version} installed successfully"
+        separator = "=" * len(message)
+        self.stdout.write(self.HEADING(f"{separator}\n{message}\n{separator}"))
+        self.stdout.write(f"""
+Congratulations! You have successfully installed django CMS,
+the lean enterprise content management powered by Django!
+
+Now, to start the development server first go to your newly
+created project:
+$ {self.style.SUCCESS("cd " + project)}
+$ {self.style.SUCCESS("./manage.py runserver")}
```

Comment:
```suggestion
$ {self.style.SUCCESS("python -m manage runserver")}
```

This would be my preference when suggesting commands to people. Help start good habits for them.

Reply:
It's `python manage.py runserver`, isn't it? There is no `manage` module, or is there?

Reply:
`-m` invokes the command as a module so you drop the `.py`

Reply:
Yes, but I doubt there is a `manage` module.